src/amo/components/Addon/index.js

```diff
@@ -545,6 +562,8 @@ export class AddonBase extends React.Component {
 
           {this.renderRatingsCard()}
 
+          {this.renderContributeCard()}
```

Comment:
If the `Contribute` component accepted just `addon` as previously suggested this wouldn't need to be a method with logic but a simple tag: `<Contribute addon={addon} />`.